test/functional/test_runner.py

```diff
@@ -197,6 +197,7 @@
     'feature_minchainwork.py',
     'rpc_estimatefee.py',
     'rpc_getblockstats.py',
+    'rpc_getbalances.py',
```

Comment:
Note that test/functional/wallet_balance.py already exists and these tests could be placed there, particularly as any of the tests in that file that don't call getbalances yet should be converted to it (see #18451 which does that).

Reply:
Thanks!
Have update the test and removed the now redundant rpc_getbalance.